parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -16,6 +16,7 @@ KUBERNETES_VERSION={{GetParameter "kubernetesVersion"}}
 HYPERKUBE_URL={{GetParameter "kubernetesHyperkubeSpec"}}
 KUBE_BINARY_URL={{GetParameter "kubeBinaryURL"}}
 CUSTOM_KUBE_BINARY_URL={{GetParameter "customKubeBinaryURL"}}
+PRIVATE_KUBE_BINARY_URL="{{GetLinuxPrivatePackageURL}}"
```

Comment:
@Bryce-Soghigian @tallaxes for a new addition to ce_cmd.sh

for @daxakp, is this URL something that needs to be provided through RP? can we not hardcode/program it into the VHD?

we are trying to get away from having URLs in the contract as much as possible.

Reply:
is that possible to reuse the `CUSTOM_KUBE_BINARY_URL`?

Reply:
Yes, @anujmaheshwari1, it needs to be provided from RP, to decide which cached package to use from the VHD at run-time.

Reply:
@bcho, others can confirm but as i understand, custom_url is not used except rarely for internal testing? Private_url naming etc. is in sync with windows naming, potentially that can be used instead of custom_url and can remove custom_url.